test/ee2_scheduler_kafka_test.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""Module to test out logging to kafka."""
+
+import json
+import logging
+
+from execution_engine2.utils.KafkaUtils import (
+    KafkaHandler,
+    send_message_to_kafka,
+    _test_sample_consumer,
+)
+import unittest
+
+
+class ExecutionEngine2SchedulerTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """
```

Comment:
You could remove the setUpClass method and put this docstring at the top of the class